engines/wintermute/ad/ad_scene.cpp

```diff
@@ -1154,8 +1154,9 @@ bool AdScene::traverseNodes(bool doUpdate) {
 
 #ifdef ENABLE_WME3D
 		if (!doUpdate && _sceneGeometry && _layers[j]->_main) {
-			// TODO: render shadow geometry in case of
-			// shadow type being stencil
+			if (_gameRef->getMaxShadownType(nullptr) >= SHADOW_STENCIL) {
```

Comment:
Is this supposed to be "ShadownType", or is that n a typo?

Reply:
typo fixed